languages/en.php

```diff
@@ -154,6 +154,7 @@
 	'access:write' => "Write access",
 	'access:admin_only' => "Administrators only",
 	'access:missing_name' => "Missing access level name",
+	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
```

Comment:
Do we need the above warning, that the access levels of comments will also
change when owner change the entity access level? This is the expected
behavior no?
On Mar 20, 2015 2:16 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In languages/en.php
> <https://github.com/Elgg/Elgg/pull/8088#discussion_r26825268>:
>
> > @@ -154,6 +154,7 @@
> >  	'access:write' => "Write access",
> >  	'access:admin_only' => "Administrators only",
> >  	'access:missing_name' => "Missing access level name",
> > +	'access:comments:change' => "There are comments on this content, by changing the access level you will also be changing the access of those comments.  Do you want to continue?",
>
> The message sounds like it's being aimed for developers. It might be
> better to use "visibility" instead of "access level". Then again we do use
> "access" also as the input label.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8088/files#r26825268>.
>